### PR TITLE
office-js: add OfficeThemeChanged event

### PR DESCRIPTION
**Pull request by ElizabethSamuel-MSFT**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [ ] Test the change in your own code. (Compile and run.)
- [ ] Add or edit tests to reflect the change. (Run with `npm test`.)
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] Run `npm run lint package-name` (or `tsc` if no `tslint.json` is present).

Select one of these and delete the others:

If changing an existing definition:
- [ ] Provide a URL to documentation or source code which provides context for the suggested changes: <<url here>>
- [ ] Increase the version number in the header if appropriate.
- [ ] If you are making substantial changes, consider adding a `tslint.json` containing `{ "extends": "dtslint/dt.json" }`.

### CODE REVIEW

File: types/office-js/index.d.ts [hunk 1]
```diff
@@ -1974,6 +1974,14 @@ declare namespace Office {
          * Triggers when a customXmlPart node was replaced.
          */
         NodeReplaced,
+        /**
+         * Triggers when the OfficeTheme was changed in Outlook.
```
Comment (AlexJerabek) [inline types/office-js/index.d.ts]: "was changed" -> "changes"

File: types/office-js/index.d.ts [hunk 2]
```diff
@@ -14782,9 +14790,7 @@ declare namespace Office {
         /**
          * Adds an event handler for a supported event.
          *
-         * Currently the only supported event type is Office.EventType.ItemChanged, which is invoked when the user selects a new item. 
-         * This event is used by add-ins that implement a pinnable taskpane, and allows the add-in to refresh the taskpane UI based on the currently 
-         * selected item.
+         * Currently the supported event types are `Office.EventType.ItemChanged` and `Office.EventType.OfficeThemeChanged`.
```
Comment (AlexJerabek) [inline types/office-js/index.d.ts]: Either "Currently, the supported event types..." or "The currently supported event types..."

**Review by AlexJerabek — CHANGES_REQUESTED** [review]
Some small grammar change requests.

File: types/office-js/index.d.ts (hunk 1 above)
Comment (ElizabethSamuel-MSFT) [reply types/office-js/index.d.ts]: Per our discussion, updated to "is changed"

**Review by AlexJerabek — APPROVED** [review]

**Pull request merged**